### PR TITLE
[macropad] doodle - 03/14

### PR DESCRIPTION
**Pull request by AlisGraveNil**
## Types of Changes

- [ ] Core
- [ ] Bugfix
- [ ] New feature
- [ ] Enhancement/optimization
- [X] Keyboard (addition or update)
- [X] Keymap/layout/userspace (addition or update)
- [ ] Documentation

## Issues Fixed or Closed by This PR

* 

## Checklist

- [x] My code follows the code style of this project: [**C**](https://docs.qmk.fm/#/coding_conventions_c), [**Python**](https://docs.qmk.fm/#/coding_conventions_python)
- [X] I have read the [**PR Checklist** document](https://docs.qmk.fm/#/pr_checklist) and have made the appropriate changes.
- [X] My change requires a change to the documentation.
- [X] I have updated the documentation accordingly.
- [X] I have read the [**CONTRIBUTING** document](https://docs.qmk.fm/#/contributing).
- [X] I have added tests to cover my changes.
- [X] I have tested the changes and verified that they work and don't break anything (as well as I can manage).

### CODE REVIEW

**Review by drashna — APPROVED** [review]

File: keyboards/clawsome/doodle/keymaps/default/keymap.c [hunk 1]
```diff
@@ -0,0 +1,26 @@
...
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with this program.  If not, see <http://www.gnu.org/licenses/>.
+ */
+
+#include QMK_KEYBOARD_H
+
+const uint16_t PROGMEM keymaps[][MATRIX_ROWS][MATRIX_COLS] = {
+    [0] = LAYOUT( \
```
Comment (fauxpark) [inline keyboards/clawsome/doodle/keymaps/default/keymap.c:20]:
```suggestion
    [0] = LAYOUT(
```

**Review by fauxpark — CHANGES_REQUESTED** [review]

**Review by fauxpark — APPROVED** [review]

**Comment by fauxpark** [comment]
Thanks!

File: keyboards/clawsome/doodle/doodle.h [hunk 2]
```diff
@@ -0,0 +1,27 @@
...
+
+#include "quantum.h"
+
+#define LAYOUT( \
+    K00, K01, K02,   \
+    K10, K11, K12    \
+) { \
+    { K00,   K01,   K02   }, \
+    { K10,   K11,   K12   }, \
+}
```
Comment (fauxpark) [inline keyboards/clawsome/doodle/doodle.h:27]:
```suggestion
}
```

**Review by fauxpark — CHANGES_REQUESTED** [review]
Oops, one last thing.

**Review by drashna — APPROVED** [review]

**Comment by AlisGraveNil** [comment]
@fauxpark - anything else I need to change to get final approval for this?

**Review by fauxpark — APPROVED** [review]

**Review by drashna — APPROVED** [review]

**Comment by drashna** [comment]
Thanks!

File: keyboards/clawsome/doodle/doodle.h [hunk 3]
```diff
@@ -0,0 +1,27 @@
...
+#pragma once
+
+#include "quantum.h"
+
+#define LAYOUT( \
+    K00, K01, K02,   \
+    K10, K11, K12    \
+) { \
+    { K00,   K01,   K02   }, \
+    { K10,   K11,   K12   }, \
```
Comment (drashna) [inline keyboards/clawsome/doodle/doodle.h:26]:
```suggestion
    { K10,   K11,   K12   } \
```

**Pull request merged**